tests/integration/multipass/test_multipass_provider.py

```diff
@@ -43,15 +45,16 @@ def test_create_environment(installed_multipass, instance_name):
     assert test_instance.exists() is False
 
 
-@pytest.mark.parametrize(
-    "alias",
-    set(BuilddBaseAlias)
-    # skip devel images because they are not available on macos
-    - {BuilddBaseAlias.XENIAL, BuilddBaseAlias.LUNAR, BuilddBaseAlias.DEVEL},
-)
+@pytest.mark.parametrize("alias", set(BuilddBaseAlias) - {BuilddBaseAlias.XENIAL})
 def test_launched_environment(alias, installed_multipass, instance_name, tmp_path):
     """Verify `launched_environment()` creates and starts an instance then stops
     the instance when the method loses context."""
+    if sys.platform == "darwin" and alias == BuilddBaseAlias.KINETIC:
```

Comment:
I have like 3 tools in my pytest toolkit and am slowly adding them as I work with you and Tiago.  Let me try adding the `skipif` decorator on the main branch in another PR.

Reply:
Oh I remember now.  This was the simplest and most readable solution.

`@skipif` can't read data from the `parameterize` decorator.  And I couldn't figure out how to mark individual parameters like `pytest.param(<value>, marks=pytest.mark.skipif(...))` for this test, since I'm creating the parameters from a set.

Any other solution required making a class and adding complexity.